CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(Mesh3DProcrustesAlignFilter)
+
+set(Mesh3DProcrustesAlignFilter_LIBRARIES Mesh3DProcrustesAlignFilter)
```

Comment:
Since we are not generating a library (there is no *src/* directory, this module is header-only), remove this line. This will address:

> /usr/bin/ld: cannot find -lMesh3DProcrustesAlignFilter